src/models/db.js

```diff
@@ -27,7 +27,16 @@ let deferreds = [];
 let state = IDLE;
 let db    = null;
 
-module.exports = () => {
+const { name, user, password, host, port } = config.get('db');
+
+const sequelize = new Sequelize(name, user, password, {
+  host,
+  port,
+  dialect: 'postgres',
+  logging: false
+});
```

Comment:
I don't think this makes a big perf difference, as I believe what makes the big perf hit is loading all the model and calling `sync`